test/unit/core.js

```diff
@@ -197,6 +197,34 @@ QUnit.test( "globalEval execution after script injection (#7862)", function( ass
 	assert.ok( window.strictEvalTest - now < 500, "Code executed synchronously" );
 } );
 
+QUnit.test( "jQuery.append with crossOrigin attribute", function( assert ) {
+	var done = assert.async();
+	assert.expect( 1 );
+
+	var timeout = null;
+
+	Globals.register( "corsCallback" );
+	window.corsCallback = function( response ) {
+		assert.ok( response === true, "Cors" );
+
+		if ( timeout != null ) {
+			window.clearTimeout( timeout );
+		}
+
+		done();
+	};
+
+	var src = baseURL + "mock.php?action=cors";
```

Comment:
We typically put all var declarations near the top of the function under one `var` keyword, one declaration per line if immediately assigned, unassigned vars all in one line (the first one). Please see other tests for comparison.

Here I'd expect `assert.expect( 1 );` to be the first line, then an empty line & then all vars declared with one var statement.